libs/search/src/lib/state/reducer.ts

```diff
@@ -125,6 +125,15 @@ export function reducerSearch(
         },
       }
     }
+    case fromActions.CLEAR_PAGINATION:
+      return {
+        ...state,
+        params: {
+          ...state.params,
+          from: 0,
+          size: state.params.size,
+        },
```

Comment:
```suggestion
        params: {
          ...state.params,
          from: 0,
        },
```
size should be already good with spread operator